fbw-a380x/src/systems/instruments/src/MFD/FMC/FlightManagementComputer.ts

```diff
@@ -139,6 +151,23 @@ export class FlightManagementComputer implements FmcInterface {
   // TODO remove this cyclic dependency, isWaypointInUse should be moved to DataInterface
   private dataManager: DataManager | null = null;
 
+  private readonly flightPhase = ConsumerSubject.create<FmgcFlightPhase>(null, this.flightPhaseManager.phase);
```

Comment:
```suggestion
  private readonly flightPhase = ConsumerSubject.create<FmgcFlightPhase>(this.sub.on('fmgc_flight_phase'), this.flightPhaseManager.phase);
```
You can do this, if you pull `private sub = this.bus.getSubscriber<FlightPhaseManagerEvents & MfdUIData>()`
to the class methods, saves you two lines of `setConsumer` calls